src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrUpdater.cs

```diff
@@ -227,7 +242,9 @@ public async Task UpdateVmr(SourceMapping mapping, string? targetRevision, bool
         }
 
         await TagRepo(mapping, toRevision);
-
+        cancellationToken.ThrowIfCancellationRequested();
+        await ApplyVmrPatches(mapping, cancellationToken);
+        cancellationToken.ThrowIfCancellationRequested();
```

Comment:
Should this line be duplicated before and after the async task?

Reply:
Yes, we want to check after each operation whether we're not being cancelled

Reply:
My bad, I did not notice the async task on line 244